openapi-docs/docs/MiscApi.md

```diff
@@ -0,0 +1,224 @@
+# AmbianicFunctionsCollection.MiscApi
+
```

Comment:
@vickywane What is the purpose of this MiscAPI file?

Reply:
It is a duplicate and it has been removed. The `openapi-generator` sometimes generates duplicate files if there is a slight change in the specification file.